### ISSUE TITLE
Add classifiers to setup.py

### ISSUE THREAD

**Issue by Zopieux** [aitjcize/cppman#36]
Please add the right classifiers to `setup.py`. I would do it myself and open a PR but I have no idea what is the stability level or the exhaustive list of supported Python versions.

Given the Travis build, I suggest at least the following:

```python
classifiers = (
    'Programming Language :: Python :: 2.6'
    'Programming Language :: Python :: 2.7',
    'Programming Language :: Python :: 2 :: Only',
)
```
Of course, if it is Python 3 compatible, some other should be added and `2 :: Only` be removed.

**Comment by aitjcize** [comment]
cppman currently does not support python3, but I have plan to port it to python3 eventually. Thanks for the suggestion, can you send a PR to add the classifiers? Thanks a lot!

**Comment by Zopieux** [comment]
Implemented in #37.
